helios-client/src/main/java/com/spotify/helios/client/Response.java

```diff
@@ -17,12 +17,12 @@
 
 package com.spotify.helios.client;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
Sorry. I would appreciate any tips on making IntelliJ less crap at managing imports. I'll disable my import-optimisation for now.